code/game/gamemodes/shadowling/shadow_abilities.dm

```diff
@@ -0,0 +1,542 @@
+	var/mob/M
+
+	to_chat(user, "<span class='shadowling'><b>You focus your telepathic energies abound, harnessing and drawing together the strength of your thralls.</b></span>")
+
+	for(var/_M in get_antagonists(ANTAG_DATUM_THRALL))
+		var/mob/M = _M
+		thralls++
+		to_chat(M, "<span class='shadowling'>You feel hooks sink into your mind and pull.</span>")
+
+
+	if(!do_after(user, 30, target = user))
+		to_chat(user, "<span class='warning'>Your concentration has been broken. The mental hooks you have sent out now retract into your mind.</span>")
+		return
+
+	if(thralls >= CEILING(3*SSticker.mode.thrall_ratio, 1) && !screech_acquired)
+		screech_acquired = TRUE
+		to_chat(user, "<span class='shadowling'><i>The power of your thralls has granted you the <b>Sonic Screech</b> ability. This ability will shatter nearby windows and deafen enemies, plus stunning silicon lifeforms.</span>")
+		user.mind.AddSpell(new /obj/effect/proc_holder/spell/unearthly_screech(null))
+
+	if(thralls >= CEILING(7*SSticker.mode.thrall_ratio, 1) && !blind_smoke_acquired)
+		blind_smoke_acquired = TRUE
+		to_chat(user, "<span class='shadowling'><i>The power of your thralls has granted you the <b>Blinding Smoke</b> ability. It will create a choking cloud that will blind any non-thralls who enter. \
+			</i></span>")
+		user.mind.AddSpell(new /obj/effect/proc_holder/spell/blindness_smoke(null))
+
+
+	if(thralls < victory_threshold)
+		to_chat(user, "<span class='shadowling'>You do not have the power to ascend. You require [victory_threshold] thralls, but only [thralls] living thralls are present.</span>")
+
+	else if(thralls >= victory_threshold)
+		to_chat(user, "<span class='shadowling'><b>You are now powerful enough to ascend. Use the Ascendance ability when you are ready. <i>This will kill all of your thralls.</i></span>")
+		to_chat(user, "<span class='shadowling'><b>You may find Ascendance in the Shadowling Evolution tab.</b></span>")
+		for(M in GLOB.alive_mob_list)
+			if(is_shadow(M))
+				var/obj/effect/proc_holder/spell/self/collective_mind/CM
+				if(CM in M.mind.spell_list)
+					M.mind.spell_list -= CM
+					qdel(CM)
+				M.mind.RemoveSpell(/obj/effect/proc_holder/spell/self/shadowling_hatch)
+				M.mind.AddSpell(new /obj/effect/proc_holder/spell/self/shadowling_ascend(null))
+				if(M == user)
+					to_chat(user, "<span class='shadowling'><i>You project this power to the rest of the shadowlings.</i></span>")
+				else
+					to_chat(M, "<span class='shadowling'><b>[user.real_name] has coalesced the strength of the thralls. You can draw upon it at any time to ascend. (Shadowling Evolution Tab)</b></span>") //Tells all the other shadowlings
+
+
+
+///////////////////
+////BLIND SMOKE////
+///////////////////
+
+/obj/effect/proc_holder/spell/blindness_smoke //Spawns a cloud of smoke that blinds non-thralls/shadows and grants slight healing to shadowlings and their allies
+	name = "Blindness Smoke"
+	desc = "Spews a cloud of smoke which will blind enemies."
```

Comment:
See Flash Freeze. This is an escape ability, and going overboard will only make history repeat.